notebooks/EDA Climate.ipynb

```diff
@@ -0,0 +1,1821 @@
+{
```

Comment:
<p>Put these two plots next to eachother on the same subplot (use ax kwarg)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/laos-climate-change/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>